docs/guide/roi.rst

```diff
@@ -1,6 +1,6 @@
-==================
-Region of Interest
-==================
+===================
+Regions of Interest
+===================
```

Comment:
I think this should be reverted.

Reply:
Is that because the ``sunpy.roi.roi.roi`` objects are for a single region of interest? Even in this case I feel the pluralization is correct because the module can be used to create an arbitrary number of ROIs.

Reply:
Sure.